http-svr/xmlrpc_client.ml

```diff
@@ -155,7 +155,7 @@ let assert_dest_is_ok host =
 
 (** Returns an stunnel, either from the persistent cache or a fresh one which
     has been checked out and guaranteed to work. *)
-let with_reusable_stunnel ?use_fork_exec_helper ?write_to_log ?verify_cert host port f =
+let with_reusable_stunnel ?use_fork_exec_helper ?write_to_log host port f =
```

Comment:
I though I removed all of these verify cert params in this commit https://github.com/xapi-project/xen-api-libs-transitional/commit/1a2e4ba1f43773d1939f6c3f36dc21e2ba4b1d2b but apparently not!